js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
+let homeItem = document.createElement('a');
+let dontClickItem = document.createElement('a');
+dontClickItem.style.color = 'green';
+
+homeItem.textContent = 'Home';
+homeItem.style.color = 'green';
+navItems[0].textContent = 'Services';
+navItems[1].textContent = 'Product';
+navItems[2].textContent = 'Vision';
+navItems[3].textContent = 'Features';
+navItems[4].textContent = 'About';
+navItems[5].textContent = 'Contact';
+dontClickItem.textContent = 'Dont Click';
+navBar.appendChild(dontClickItem);
+navBar.prepend(homeItem);
+
+navItems.forEach(function (ele) {
+  ele.style.color = 'green';
+})
+
+let h1Item = document.querySelector('h1');
+// let br = document.createElement('BR');
+// h1Item.textContent = 'DOM ' + document.write('<br> /') + 'IS AWESOME';
+h1Item.innerHTML = 'DOM <br /> IS <br /> AWESOME';
+let buttonOne = document.querySelector('button');
+buttonOne.textContent = 'Get Started';
+
+let headerIMG = document.getElementById('cta-img');
+headerIMG.src = 'img/header-img.png';
+
+let mainContent = document.querySelectorAll('.text-content');
+mainContent[0].firstChild.textContent = 'Features';
+mainContent[0].lastChild.textContent = 'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
+mainContent[1].firstChild.textContent = 'About';
+mainContent[1].lastChild.textContent = 'About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
+mainContent[2].firstChild.textContent = 'Services';
+mainContent[2].lastChild.textContent = 'Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
+mainContent[3].firstChild.textContent = 'Product';
+mainContent[3].lastChild.textContent = 'Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
+mainContent[4].firstChild.textContent = 'Vision';
+mainContent[4].lastChild.textContent = 'Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.';
+
+let mainIMG = document.getElementById('middle-img');
+mainIMG.src = 'img/mid-page-accent.jpg';
+
+let contactInfo = document.querySelector('.contact');
+let contactH4 = document.createElement('h4');
+contactH4.textContent = 'Contact';
+let contactAddress = document.createElement('p');
+contactAddress.innerHTML = '123 Way 456 Street, <br /> Somewhere, USA';
```

Comment:
the innerHTML method has been deprecated because of a security vulnerability. highly recommended to never use this method in production code.